static/app/views/issueList/draggableTabBar.tsx

```diff
@@ -130,18 +161,18 @@ export function DraggableTabBar({
         onSelectionChange={setSelectedTabKey}
         selectedKey={selectedTabKey}
         showTempTab={showTempTab}
-        onAddView={onAddView}
+        onAddView={e => handleOnAddView(e)}
         orientation="horizontal"
       >
-        {tabs.map(tab => (
+        {[...tabs, tempTab].map(tab => (
```

Comment:
This might be a little controversial - I was deciding between two options to handle temporary tabs:
1. Pass in the temp tab as its own prop. It is not included in the top level tabs state, but it is included in the aria collection so it still has accessibly features. (The current implementation)
2. Have the temporary tab be passed in with the `tabs` prop, and make a note that the temp tab must have `key = 'temporary-tab'`.

Number 1 seemed to be more intuitive and less error prone than number 2, so that's what I went with. But pls lmk if there might be a better design than this.

Reply:
I agree that separating the temp tab from the list of saved views makes sense and is probably the better way to go here 👍. Conceptually it is very different and has different behavior.